data/route_goldenfiles/krakow_center.txt

```diff
@@ -1,3 +1,3 @@
 https://www.openstreetmap.org/relation/2053781 from Lane #4489 to Some(LaneID(8696))
   Politechnika: Position(Lane #4489, 68.497m) driving, Position(Lane #8197, 15.9997m) sidewalk
```

Comment:
They're a change detector to notice if some importing change totally breaks bus routes. The screenshot diff test is a nice way to visually spot changes to the map, and this is the equivalent for bus routes. An interesting diff here would be stops or routes disappearing. Lane IDs changing isn't too interesting.

Reply:
(Diff testing and goldenfiles have been my rough approach to testing the map importing process. Because importing has never been totally "correct" and because the data itself is often pretty bad, it requires human judgment to know if something is importing well or not. After judging the initial state of a map, quickly glancing at diffs to that map is a pretty easy way to spot regressions or verify fixes.)